highfidelity/hifi
# Momentum aka flickable for WebView

https://highfidelity.fogbugz.com/f/cases/6610/Add-Momentum-to-Scrolling

Test plan
- open different WebView involved apps and scripts
- make sure everything works as expected
